test/fixtures/basic/pages/custom-extension.md

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Nice idea BTW. But is this a valid usage? 😆

Reply:
Definitely not for a md lol, but the hypothetical custom loader should convert it to raw Vue Component. It's a simple test, but I guess I could add the loader as well :)

Reply:
I don't understand how it works without defining a dummy loader in the nuxt.config.js though?

Reply:
is requiring a file that doesn't match anything in webpack mean just passing the text through?

Reply:
It doesn't, I have left apart the whole loader logic because the main idea is to support custom extensions.